deps/llvm.mk

```diff
@@ -336,6 +336,7 @@ endif
 ifneq ($(LLVM_LLDB_TAR),)
 	$(JLCHECKSUM) $(LLVM_LLDB_TAR)
 endif
+	-rm -rf $(LLVM_SRC_DIR)
```

Comment:
Someone could touch this build target manually or we could revert to testing for configure. But from the time stamp on cmakelist, we can't tell if the user has a good or corrupted build.